.gitignore

```diff
@@ -4,6 +4,5 @@
 deps/deps.jl
 Manifest.toml
 *.png
-docs/build
-.vscode
-.DS_store
```

Comment:
why remove this?

Reply:
I was told in a separate PR that such things (anything not generated by the package itself) should go into my global `.gitignore` instead of the package's one. I checked and it was me who added these back in November. But I can leave them in if you prefer